src/Test.Utilities/AdditionalMetadataReferences.cs

```diff
@@ -9,6 +10,13 @@ namespace Test.Utilities
 {
     public static class AdditionalMetadataReferences
     {
+        public static ReferenceAssemblies Net80 { get; } = new ReferenceAssemblies(
+            "net8.0",
+            new PackageIdentity(
+                "Microsoft.NETCore.App.Ref",
+                "8.0.0-preview.6.23329.7"),
```

Comment:
Does this need to be updated whenever the repo updates the version it depends on preview to preview?

Reply:
Not really. Normally the signatures in the reference assemblies don't change to the point of affecting the unit tests so they just keep working. Plus, it'll end up in dotnet/roslyn-sdk soon.

Reply:
Another potential approach we take for tests is to add an additional source file to the test with custom stub implementation of the newer APIs added to the Framework which are not yet available via the currently referenced assemblies. We can do a sweeping cleanup later once we move the tests to reference packages with required APIs.

Reply:
> Another potential approach we take for tests is to add an additional source file to the test with custom stub implementation of the newer APIs

This is necessary when the APIs haven't been published, but in this case we can reference the real APIs and avoid the risk involved in cases where a deviation in our test implementation impacts the behavior of the analyzer.